Dockerfile

```diff
@@ -8,40 +8,36 @@ LABEL build_version="Linuxserver.io version:- ${VERSION} Build-date:- ${BUILD_DA
 LABEL maintainer="nemchik"
 
 # environment settings
+ARG DEBIAN_FRONTEND="noninteractive"
 ENV NZBHYDRA2_RELEASE_TYPE="Release"
 
 RUN \
   echo "**** install packages ****" && \
-  apk add -U --no-cache --virtual=build-dependencies \
-    unzip && \
-  apk add -U --no-cache \
+  apt-get update && \
+  apt-get install --no-install-recommends -y \
     curl \
     jq \
-    openjdk11-jre-headless \
-    python3 && \
+    openjdk-11-jre-headless \
+    python3 \
+    unzip && \
   echo "**** install nzbhydra2 ****" && \
   if [ -z ${NZBHYDRA2_RELEASE+x} ]; then \
-    NZBHYDRA2_RELEASE=$(curl -sX GET "https://api.github.com/repos/theotherp/nzbhydra2/releases/latest" \
-    | jq -r .tag_name); \
+  NZBHYDRA2_RELEASE=$(curl -sX GET "https://api.github.com/repos/theotherp/nzbhydra2/releases/latest" \
+  | jq -r .tag_name); \
   fi && \
   NZBHYDRA2_VER=${NZBHYDRA2_RELEASE#v} && \
   curl -o \
-  /tmp/nzbhydra2.zip -L \
+    /tmp/nzbhydra2.zip -L \
     "https://github.com/theotherp/nzbhydra2/releases/download/v${NZBHYDRA2_VER}/nzbhydra2-${NZBHYDRA2_VER}-linux.zip" && \
   mkdir -p /app/nzbhydra2/bin && \
   unzip /tmp/nzbhydra2.zip -d /app/nzbhydra2/bin && \
-  rm -rf \
-    /app/nzbhydra2/bin/upstart/ \
-    /app/nzbhydra2/bin/rc.d/ \
-    /app/nzbhydra2/bin/sysv/ \
-    /app/nzbhydra2/bin/systemd/ && \
   chmod +x /app/nzbhydra2/bin/nzbhydra2wrapperPy3.py && \
-  echo -e "ReleaseType=${NZBHYDRA2_RELEASE_TYPE}\nPackageVersion=${VERSION}\nPackageAuthor=linuxserver.io" > /app/nzbhydra2/package_info && \
+  echo "ReleaseType=${NZBHYDRA2_RELEASE_TYPE}\nPackageVersion=${VERSION}\nPackageAuthor=linuxserver.io" > /app/nzbhydra2/package_info && \
```

Comment:
doesn't `echo` require `-e` to interpret variables?

Reply:
Not with Ubuntu, only Alpine - at least for the purposes of Dockerfiles